pkg/distributor/distributor.go

```diff
@@ -467,6 +470,11 @@ func New(cfg Config, clientConfig ingester_client.Config, limits *validation.Ove
 			Help: "Total number of label values with newlines seen at ingestion time.",
 		}, []string{"user"}),
 
+		droppedNativeHistograms: promauto.With(reg).NewCounterVec(prometheus.CounterOpts{
+			Name: "cortex_distributor_dropped_native_histograms_total",
```

Comment:
there's also `cortex_discarded_samples_total{user=, reason=}` if you want to use that. It's already displayed in the tenants dashboard

Reply:
I don't really want to use that as all of those return 4xx so this would be a major inconsistency I think.